contracts/neutron_interchain_txs/src/contract.rs

```diff
@@ -295,7 +297,7 @@ fn execute_undelegate(
 }
 
 fn execute_clean_ack_results(deps: DepsMut) -> StdResult<Response<NeutronMsg>> {
-    let keys: Vec<StdResult<String>> = ACKNOWLEDGEMENT_RESULTS
+    let keys: Vec<StdResult<(String, u64)>> = ACKNOWLEDGEMENT_RESULTS
```

Comment:
You can use `Vec<_>` instead
```suggestion
    let keys: Vec<_> = ACKNOWLEDGEMENT_RESULTS
```